serverless.yaml

```diff
@@ -270,6 +195,7 @@ resources:
         DeletionPolicy: Retain
         UpdateReplacePolicy: Retain
         Properties:
+          TableName: '${self:custom.resourceTableName}'

```

Comment:
Changing the TableName requires replacement of the ddb table. Is that what we want?

** I'm sure it requires replacement when updating the name. I'm not 100% sure about what happens when changing from some table name to not specifying any name, but I'd expect it to be replaced.

Reply:
Good call out I am actually just moving it from below in the properties; no functional change here

Reply:
Right. My bad